abtesting/ABTestingExample/README.md

```diff
@@ -0,0 +1,25 @@
+A/B Testing
+========
+
+Firebase A/B Testing leverages Remote Config to automatically trial new app
+behaviors and exports the experiment results into Firebase Analytics. This
+sample demonstrates using an A/B test to test mutiple color schemes within
+an app.
+
+## Quickstart Setup
+
+In Firebase Console, create a new A/B test with any name. In step two under
+'Variants', click the 'Add Parameter' button and create a parameter named
+`color_scheme`. Set its default value in the control group to `default` and
+its value in 'Variant A' to `dark`. In the third step, select any goal metric.
+
+Verify the following two flows:
```

Comment:
nit: maybe "one of the following two flows", since they're redundant

Reply:
I think "following two flows" refers to the "Test on device" and "Published experiment" sections rather than installation ID/installation auth token (and only installation auth token remains on the SwiftUI version). Feel free to clarify if I misunderstood 👍

Reply:
SGTM